src/main/java/racing/model/Decider.java

```diff
@@ -0,0 +1,51 @@
+package racing.model;
+
+import racing.util.GetRandomNumber;
+import racing.view.GameView;
+
+import java.util.*;
+
+public class Decider {
+    GetRandomNumber getRandomNumber = new GetRandomNumber();
+    GameView gameView = new GameView();
```

Comment:
필드들에 private 접근 제어자들을 추가해주세요.
```suggestion
    private final GameView gameView = new GameView();
```